src/main/java/de/uni_tuebingen/ub/nppm/model/SelektionProvenance.java

```diff
@@ -0,0 +1,31 @@
+package de.uni_tuebingen.ub.nppm.model;
+
+import javax.persistence.*;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+@MappedSuperclass
+@Cacheable
+@org.hibernate.annotations.Cache(usage = CacheConcurrencyStrategy.READ_WRITE)
+public abstract class SelektionProvenance extends SelektionBezeichnung {
```

Comment:
It's basically OK to introduce a new class, but since we really want to have provenance on all types of selections (e.g. to show them all in the Administration backend) we might also just add the new fields directly to Selektion.java.

Reply:
The problem is here that there are some Selektions Entities with no provenance information. Like SelektionAmtWeiheGastEinzelbeleg , SelektionAmtWeiheGastPerson and SelektionQuellengattungGast...

Reply:
Ah, alright! Then let's just keep it this way, thanks!